lib/Imagine/Filter/Transformation.php

```diff
@@ -96,7 +96,7 @@ public function getFilters()
     {
         if (null === $this->sorted) {
             ksort($this->filters);
-            $this->sorted = call_user_func_array('array_merge', $this->filters);
+            $this->sorted = array_merge($this->filters);
```

Comment:
@romainneutron I would use an explicit empty array in the ``else`` clause rather than ``$this->filters``. It will be easier to understand